docs/en/getting-started/create-or-restore-your-status-profile-using-a-recovery-phrase.md

```diff
@@ -1,16 +1,97 @@
 ---
 id: 374
 revision: 0
+revision: o
 language: en
 hide:
   - navigation
 ---
 
 # Create or restore your Status profile using a recovery phrase
 
- :octicons-tools-24: In progress
+You can link your existing Ethereum address to a new Status profile. For instance, if you already have a [non-custodial Ethereum wallet][how-to-use-wallet-your-quick-start-guide] such as Metamask or Trust Wallet, you already have an Ethereum address and a [recovery phrase][understand-your-status-keys-and-recovery-phrase].
```

Comment:
This paragraph only introduces "create" but not "restore". I would add a sentence "If you already have a Status account and you used your profile in the past 30 days, you can restore it with the profile's recovery phrase.".

Reply:
Suggestion included.